src/data-source/esi/EsiAsset.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * ESI format for assets returned by character and corporation asset
+ * endpoints.
+ */
+export interface EsiAsset {
+  is_blueprint_copy?: boolean;
```

Comment:
Whis does one of these have a question mark?

Reply:
It's typescript's syntax for optional properties.